src/vmware/azext_vmware/action.py

```diff
@@ -0,0 +1,69 @@
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long, protected-access, too-few-public-methods
+
+import argparse
+from typing import Dict, List
+from knack.util import CLIError
+from azext_vmware.vendored_sdks.avs_client.models import ScriptExecutionParameter, ScriptExecutionParameterType, ScriptStringExecutionParameter, ScriptSecureStringExecutionParameter, PSCredentialExecutionParameter
+
+
+class ScriptExecutionNamedOutputAction(argparse._AppendAction):
+
+    def __call__(self, parser, namespace, values, option_string=None):
+        namespace.named_outputs = script_execution_named_outputs(values)
+
+
+def script_execution_named_outputs(values: List[str]) -> Dict[str, str]:
+    try:
+        return dict(map(lambda x: x.split('=', 1), values))
+    except ValueError as error:
+        raise CLIError('parsing named output parameter \'{}\''.format(values)) from error
```

Comment:
Could you please use the specific error type instead of `CLIError`? For more error types, refer to doc [link](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md#error-type)